README.md

```diff
@@ -132,19 +110,39 @@ email._webidentity.mydomain.example   TXT   iss=https://issuer.example.com
 }
 ```
 
-3. Browser shows accounts metadata to autofill
+4. The browser checks if the following DNS records exists:
 
-4. User picks an johm.doe@mydomain.example.
+```
+email._webidentity.mydomain.example   TXT   iss=https://issuer.example.com
```

Comment:
The iss entry here should presumably match the registered issuer, but it doesn't. This here is an origin, while the thing that was registered was a URL with a path.

Reply:
I have similar questions as to how all the hostnames fit together. I think we want to fit into expectations that FedCM already has.

Reply:
> This here is an origin, while the thing that was registered was a URL with a path.

That's correct, but we can derive an origin from a URL, right? And wouldn't that (that origin(configURL) == origin in the DNS) suffice here?